contracts/src/models/buildings.cairo

```diff
@@ -598,17 +583,33 @@ impl BuildingCustomImpl of BuildingCustomTrait {
         destroyed_building_category
     }
 
-    fn make_payment(world: IWorldDispatcher, entity_id: ID, category: BuildingCategory, resource_type: u8) {
-        let building_config: BuildingConfig = BuildingConfigCustomImpl::get(world, category, resource_type);
+    fn pay_fixed_cost(self: Building, building_quantity: BuildingQuantityv2, world: IWorldDispatcher) {
```

Comment:
can this keep the same name as before? I feel like the function actually pays a variable cost